src/sc/sidechain.cpp

```diff
@@ -783,108 +782,91 @@ void CSidechain::UpdateScFees(const CScCertificateView& certView, int blockHeigh
         while (scFees.size() > max_size)
         {
             const auto& entry = scFees.front();
-            LogPrint("sc", "%s():%d - popping f=%d, m=%d from list, as scFees maxsize has been reached\n",
-                __func__, __LINE__, entry.forwardTxScFee, entry.mbtrTxScFee);
+            LogPrint("sc", "%s():%d - popping %s from list, as scFees maxsize has been reached\n",
+                __func__, __LINE__, entry->ToString().c_str());
```

Comment:
Here I think `.c_str()` is a bit pleonastic. Strings are already implicitly convertible to a null terminated char*

Reply:
Done